account_invoice_blocking/models/account_invoice.py

```diff
@@ -0,0 +1,84 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016 Acsone SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import api, fields, models
+
+
+class AccountInvoice(models.Model):
+
+    _inherit = 'account.invoice'
+
+    @api.model
+    def _get_move_line(self, invoice):
+        """
+        This method searches for payable or receivable move line
+        of the invoice
+        :param invoice: invoice of searched move line
+        :returns payable or receivable move line of the invoice
+        """
+        type_receivable = self.env.ref('account.data_account_type_receivable')
+        type_payable = self.env.ref('account.data_account_type_payable')
+        return self.env['account.move.line'].search(
+            [('account_id.user_type_id', 'in', [type_receivable.id,
+                                                type_payable.id]),
+             ('invoice_id', '=', invoice.id)])
+
+    @api.model
+    def _update_blocked(self, invoice, value):
```

Comment:
Please convert this method from `@api.model` to `@api.multi` without the *self recordset* argument, to follow standard changes since v10.